index.html

```diff
@@ -284,11 +302,11 @@ <h4 class="refer-link">LET'S CONNECT</h4>
                     </ul>
                 </div>
             </div>
-        </section>  
+        </section>
         <section id="contact-container">
```

Comment:
your contact section looks great in desktop size 💪, but we also need to keep mobile version consistent  please make sure that your design looks like figma template in both mobile and desktop .
**your contact me section in mobile size :**

![student contact me](https://user-images.githubusercontent.com/59707859/221055207-ebd4a782-a766-4988-b437-c15026e80bb3.PNG)